### DISCUSSION TITLE
How to set/add health to the player?

### DISCUSSION THREAD

**Discussion by Nightingale08** [Jcw87/c2-sans-fight#131] [Q&A]
i made my sans fight too hard, i would like to do this in the middle of the battle.

**Comment by ChaboCode** [comment]
I know but Im not at home rn, there is a block for every numeric thing on the game and thus for the player hp.

**Reply by Nightingale08** [reply]
alr ill wait for an answer, the sans fight is too hard i been literally trying to beat it for months now lol

this:
[timeline-shredder-1.1.0-custom-attack.zip](https://github.com/Jcw87/c2-sans-fight/files/9033093/timeline-shredder-1.1.0-custom-attack.zip)

**Reply by Nightingale08** [reply]
so how do u add/set health?

**Reply by Nightingale08** [reply]
Hello?
